### ISSUE TITLE
TypeError: forward() missing 2 required positional arguments: 'ref_imgs' and 'intrinsics'

### ISSUE THREAD

**Issue by LiangtaoZhong** [aliyun/dro-sfm#4]
Where are the 'ref_imgs and intrinsics' I need to fill in? Which file or arg?
Thanks!

**Comment by liuwb2001** [comment]
> 

I have encountered the same problem. Could you please tell me how to solve such a problem? 
Thanks!

**Comment by lovejoker1102** [comment]
Did you solve the problem??

**Comment by Ghul-huan** [comment]
yeah ,i have meet the same problem ,did anyone solve it ? it means if we only give the img to network ,it need the pose and cam intrinsic.so the infer.py have error
